docs/developer-guide/plugins.md

```diff
@@ -232,13 +232,14 @@ Validates the options for your rule.
 
 ### `stylelint.utils.checkAgainstRule`
 
-Checks CSS against a standard Stylelint rule _within your own rule_. This function provides power and flexibility for plugins authors who wish to modify, constrain, or extend the functionality of existing Stylelint rules.
+Checks CSS against a standard or custom Stylelint rule _within your own rule_. This function provides power and flexibility for plugins authors who wish to modify, constrain, or extend the functionality of existing Stylelint rules.
 
 It accepts an options object and a callback that is invoked with warnings from the specified rule. The options are:
 
 - `ruleName`: the name of the rule you are invoking
 - `ruleSettings`: settings for the rule you are invoking
 - `root`: the root node to run this rule against
+- `result?`: the postcss result for resolving and invoking custom rules
```

Comment:
[suggest] Thanks for updating the doc. Additionally, could you fix the word [for consistency](https://github.com/stylelint/stylelint/blob/35c0cfa244f725a3548e8ba884edf70c2bc67732/docs/developer-guide/plugins.md?plain=1#L70-L71)?

```suggestion
- `result?`: the PostCSS result for resolving and invoking custom rules
```